releasenotes-builder/src/main/java/com/github/checkstyle/Main.java

```diff
@@ -121,9 +106,6 @@ private static Result runGithubNotesBuilder(CliOptions cliOptions)
         if (result.hasWarnings()) {
             printListOf(result.getWarningMessages());
         }
-        if (result.hasErrors()) {
```

Comment:
We removed errors here to avoid double printing them. We may just want to consider for a future change to do the same with warnings, and then anything we call can print errors or warnings and we don't have to deal with individual lists.

Reply:
Better to deal with it later on  as problem comeup